pushmsixscript/src/pushmsixscript/microsoft_store.py

```diff
@@ -200,7 +201,7 @@ def _update_submission(config, channel, session, submission_request, headers, fi
     submission_request = str(submission_request)
     submission_request = submission_request.replace("True", "true").replace("False", "false")
     url = _store_url(config, f"{application_id}/submissions/{submission_id}")
-    response = session.put(url, submission_request, headers=headers)
+    response = session.put(url, submission_request.encode(encoding), headers=headers)
```

Comment:
json is always utf-8, can we just hardcode the encoding here and leave everything else alone?

Reply:
Is it always utf-8? rfc 7159 says it is utf-8 by default but can be utf-16 or utf-32.

I don't mind assuming utf-8 and in practice I suspect that will be fine, but this way seems possibly-safer.

Reply:
my source is wikipedia so yeah, your fix is safer :)